queue/queue.py

```diff
@@ -1,7 +1,7 @@
 from linked_list.linked_list import LinkedList
 
 
-class Queue:
+class Queue(LinkedList):
```

Comment:
One last thing, I don't think the class needs to be a sub-class of `LinkedList`. I uses the `LinkedList` class to implement it's logic, but is not a direct child.